server/applicationPattern/applicationPattern/onfModel/services/ForwardingConstructConfigurationServices.test.js

```diff
@@ -0,0 +1,241 @@
+const ForwardingConstructConfigurationServices = require('./ForwardingConstructConfigurationServices');
```

Comment:
This file only tests about half of the class. The tests do not touch methods eg. updateFcPortAsync, deleteFcPortAsync, addFcPortAsync... (run jest command with '--coverage' to see details).

Reply:
deleteFcPortAsync is still not being tested at all.